.travis.yml

```diff
@@ -13,7 +13,7 @@ branches:
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
```

Comment:
Changed this because caching `node_modules` is dangerous, e.g. it doesn't work with native addons. Caching the npm cache itself is always safe.